Project.toml

```diff
@@ -0,0 +1,22 @@
+name = "DSP"
+uuid = "717857b8-e6f2-59f4-9121-6e50c889abd2"
+
```

Comment:
Don't we need a version?

Reply:
Good catch, we do! (Well, at least once we actually want to tag the next version, but no harm in setting it to the current version now.)